config/manager/manager.yaml

```diff
@@ -35,6 +35,7 @@ spec:
             - --zap-log-level=info
             - --zap-encoder=console
             - --zap-time-encoding=rfc3339
+            - --enable-cert-rotation=true
```

Comment:
Is this another feature added in this PR? If so, let's update changelog

Reply:
No, this is part of the cert generation, moved from the webhooks to the operator for the incoming features, but this is required for this feature and used only for it atm

Reply:
I'd keep it without the external component.